### PR TITLE
[BER-FT-012020] Jacqueline C.

### PR DESCRIPTION
**Pull request by chenjacqueline**

### CODE REVIEW

File: starter-code/index.js [hunk 1]
```diff
@@ -1,17 +1,53 @@
...
-  add(item) {}
+  add(item) {
+    this.items.push(item);
+    this.items.sort((a, b) => a - b);
+    this.length = this.items.length;
+  }
 
-  get(pos) {}
+  get(pos) {
+    if (pos < this.length && pos >= 0) {
```
Comment (DalinaLeila) [inline starter-code/index.js]: 💯

File: starter-code/index.js [hunk 2]
```diff
@@ -1,17 +1,53 @@
...
+      throw new Error("OutOfBounds");
+    }
+  }
 
-  max() {}
+  max() {
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.max(...this.items);
```
Comment (DalinaLeila) [inline starter-code/index.js]: 💯

File: starter-code/index.js [hunk 3]
```diff
@@ -1,17 +1,53 @@
...
+    if (this.length === 0) {
+      throw new Error("EmptySortedList");
+    } else {
+      return Math.min(...this.items);
+    }
+  }
 
-  sum() {}
+  sum() {
+    let sum = this.items.reduce((accumulator, currentValue) => {
```
Comment (DalinaLeila) [inline starter-code/index.js]: 💯

**Comment by chenjacqueline** [comment]
Yay!! Thank you Dalina :)

**Pull request closed without merging**